run/GCHP/createRunDir.sh

```diff
@@ -284,12 +284,12 @@ fi
 
 # NOTE: Fullchem benchmarks use the climatological volcano emissions!
 if [[ "x${sim_name}" == "xfullchem" ]]; then
-    RUNDIR_VARS+="RUNDIR_VOLC_CLIMATOLOGY='\$ROOT/VOLCANO/v2021-09/so2_volcanic_emissions_CARN_v202005.degassing_only.rc'\n"
+    RUNDIR_VARS+="RUNDIR_VOLC_CLIMATOLOGY='\$ROOT/VOLCANO/v2024-03/so2_volcanic_emissions_CARN_v202202.degassing_only.rc'\n"
```

Comment:
@msulprizio: See note above about timestamp being `v202106` and not v202302`.

Reply:
Now fixed in https://github.com/geoschem/geos-chem/pull/2198/commits/bb993b2f352ac900b63b94239c26473e7da0a25c